rulesets/default.yml

```diff
@@ -51,15 +55,6 @@ Layout/LineLength:
     # Ignore RSpec description strings:
     - !ruby/regexp /^ *(context|describe|it) ['"].*['"].* do$/
   Max: 100
-Layout/MultilineAssignmentLayout:
-  SupportedTypes:
-    - block
-    - case
-    - class
-    - if
-    - kwbegin
-    - module
-    - send # This (method calls) is not in the default list.
```

Comment:
This seemed not only to apply to method calls, but also simple array and hash declarations (and probably other things).